### DISCUSSION TITLE
Simple Texture Animation in Blender

### DISCUSSION THREAD

**Discussion by giscus** [morrowind-modding/morrowind-modding.github.io#25] [General]
# Simple Texture Animation in Blender

Make a texture orbit around a sphere in this short #Blender tutorial.

https://morrowind-modding.github.io/Guides/Animation/Simple-Texture-Animation-in-Blender

**Comment by MasssiveJuice08** [comment]
See issue #24 - change references to specific hotkeys to reference the related commands instead.
